GsaGH/Helpers/Assembly/Model/Lists.cs

```diff
@@ -192,17 +191,13 @@ private void AddMemberList(GsaList list, GH_Component owner) {
     private void AddNodeList(GsaList list) {
       if (list._nodes == null || list._nodes.Count == 0) {
```

Comment:
list._nodes and list.Definition are not the same.
example in this ticket. list.Definition was "7 8 9" but list._nodes was empty

Reply:
@DominikaLos only below condition possible
1. list.Definition not empty and list._nodes=0
2. list.Definition is empty and list._node>0 ( in this case, It will create definition from node ids)
3. list.Definition is empty and list._node=0( will create empty list in GSA)

Reply:
yes, and line 192 check only list._nodes. not definition